src/main/java/org/gradle/profiler/BuildToolCommandLineScenarioDefinition.java

```diff
@@ -0,0 +1,54 @@
+package org.gradle.profiler;
+
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.function.Supplier;
+
+public abstract class BuildToolCommandLineScenarioDefinition extends ScenarioDefinition {
+    private final List<String> targets;
+    private final File toolHome;
+
+    public BuildToolCommandLineScenarioDefinition(
+        String name,
+        @Nullable String title,
```

Comment:
Why is `title` `@Nullable` here, but not in any of the subclasses?

Reply:
Fixed.